### ISSUE TITLE
digikam build is broken, via librsvg

### ISSUE THREAD

**Issue by bjornfor** [NixOS/nixpkgs#70355]
**Describe the bug**
```
$ nix-build -A digikam https://github.com/nixos/nixpkgs/archive/c75a18fea6f25dd5dfcc7d76c91cf4
9e3c53c6c5.tar.gz  # master @ today
[...]
   Compiling cssparser-macros v0.3.4
error[E0506]: cannot assign to `self.input.cached_token` because it is borrowed
   --> /build/librsvg-2.44.15/vendor/cssparser/src/parser.rs:572:17
    |
547 |     pub fn next_including_whitespace_and_comments(&mut self) -> Result<&Token<'i>, BasicParseError<'i>> {
    |                                                   - let's call the lifetime of this reference `'1`
...
560 |             Some(ref cached_token)
    |                  ---------------- borrow of `self.input.cached_token` occurs here
...
572 |                 self.input.cached_token = Some(CachedToken {
    |                 ^^^^^^^^^^^^^^^^^^^^^^^ assignment to borrowed `self.input.cached_token` occurs here
...
584 |         Ok(token)
    |         --------- returning this value requires that `self.input.cached_token.0` is borrowed for `'1`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0506`.
error: Could not compile `cssparser`.

To learn more, run the command again with --verbose.
make[2]: *** [Makefile:1934: /build/librsvg-2.44.15/target/release/librsvg_internals.a] Error 101
make[2]: Leaving directory '/build/librsvg-2.44.15'
make[1]: *** [Makefile:1438: all-recursive] Error 1
make[1]: Leaving directory '/build/librsvg-2.44.15'
make: *** [Makefile:931: all] Error 2
builder for '/nix/store/ijs8ng5xc14dc7jl0sbqbfqm8nv8i3l1-librsvg-2.44.15.drv' failed with exit code 2
cannot build derivation '/nix/store/1509dxsd0c53g9pbnjfbqmyr4hkbjic9-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/sg92idbw8rapv774g1za1r90i8wcnci8-digikam-6.2.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/sg92idbw8rapv774g1za1r90i8wcnci8-digikam-6.2.0.drv' failed
```

So digikam is broken because one of its dependencies, librsvg, is broken.

**Expected behavior**
digikam should build (and run).

**Additional info**
git bisect points to https://github.com/nixos/nixpkgs/commit/adb15c3a63fd96edcf7585394b8ccb as the first bad commit.

**Metadata**
```
$ nix run nixpkgs.nix-info -c nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.75, NixOS, 19.09beta439.b30f86ffc6a (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"nixos-19.09beta596.77b5a1965fc"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by hedning** [comment]
This should've been fixed 02585db25bc4d5d623ce083b41f727c65709ad3f

**Comment by bjornfor** [comment]
Yes, librsvg + digikam builds on master now. Thanks.
